tools/protoxform/protoxform.py

```diff
@@ -27,6 +27,11 @@
 CLANG_FORMAT_STYLE = ('{ColumnLimit: 100, SpacesInContainerLiterals: false, '
                       'AllowShortFunctionsOnASingleLine: false}')
 
+NEXT_FREE_FIELD_MIN = 3
```

Comment:
Why 3?

Reply:
Find that minimum next present before is 3, it's ok for other bound.